app/client/src/pages/common/UnsubscribeEmail.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { useParams } from "react-router-dom";
+import { unsubscribeCommentThreadAction } from "actions/commentActions";
+import { useDispatch, useSelector } from "react-redux";
+import { isUnsubscribedSelector } from "selectors/commentsSelectors";
+
+const Wrapper = styled.div`
+  height: calc(100vh - ${(props) => props.theme.headerHeight});
+  background-color: #fafafa;
+  text-align: center;
+  padding-top: calc(${(props) => props.theme.headerHeight} + 50px);
+  .bold-text {
+    font-weight: ${(props) => props.theme.fontWeights[3]};
+    font-size: 24px;
+  }
+  .button-position {
+    margin: auto;
+  }
+`;
+const UnsubscribeButton = styled.button`
+  background-color: #f3672a;
+  color: white;
+  height: 40px;
+  width: 300px;
+  border: none;
+  cursor: pointer;
+  font-weight: 600;
+  font-size: 17px;
+  margin-top: 40px;
+`;
+
+function UnsubscribeEmail() {
+  const params = useParams<{ threadId: string }>();
+  const dispatch = useDispatch();
+  const isUnsubscribed = useSelector(isUnsubscribedSelector);
+
+  const unsubscribeCommentThread = () => {
+    dispatch(unsubscribeCommentThreadAction(params.threadId));
+  };
+
+  return (
+    <Wrapper>
+      <div>
+        <p className="bold-text">Unsubscribe</p>
+        {isUnsubscribed ? (
+          <p>
+            You&apos;ve successfully unsubscribed from the corresponding comment
+            thread
+          </p>
+        ) : (
+          <>
+            <p>
+              You&apos;ll no more receive email notifications for the
+              corresponding comment thread.
```

Comment:
```suggestion
              You won't receive any more email notifications for the
              corresponding comment thread.
```
Nit: The wording was very awkward. 

Also, is it possible for us to save all these strings in the file [messages.ts](https://github.com/appsmithorg/appsmith/blob/release/app/client/src/constants/messages.ts)? Will make future i18n easier.